### PR TITLE
PAY-6453: Node 18 Upgrade Functions Node

### PR DESCRIPTION
**Pull request by Thor-tech-of-metal**
PAY-6453: Node 18 Upgrade Functions Node
    [PAY-6453] (https://tools.hmcts.net/jira/browse/PAY-6453).

### CODE REVIEW

**Review by davejones74 — APPROVED** [review]

**Review by Ollie-Smith — APPROVED** [review]
reviewed

File: charts/ccpay-callback-function/Chart.yaml [hunk 1]
```diff
@@ -25,6 +25,6 @@ dependencies:
     version: 2.0.1-beta
```
Comment (timja) [inline charts/ccpay-callback-function/Chart.yaml:25]: no harm updating this
```suggestion
    version: 2.3.4
```

File: charts/ccpay-callback-function/values.preview.template.yaml [hunk 2]
```diff
@@ -5,19 +5,20 @@ function:
...
+      key: primaryKey
+    SB_CONNECTION_STRING:
+      secretRef: fees-pay-sb-preview
       key: connectionString
-    SERVICE_CALLBACK_SUBSCRIPTION:
-      secretRef: servicebus-secret-topic-{{ .Release.Name }}-servicebus-servicecallbacktopic
-      key: subscriptionName
 servicebus:
   resourceGroup: fees-pay-rg
+  sbNamespace: "fees-pay-sb-preview"
```
Comment (timja) [inline charts/ccpay-callback-function/values.preview.template.yaml:16]: according to https://github.com/hmcts/cnp-flux-config/blob/fd9533d0f13f60d07d062ccee00a5f3186f319b2/apps/fees-pay/preview/aso/fees-pay-servicebus.yaml#L4
```suggestion
  sbNamespace: "fees-pay-servicebus-preview"
```

Comment (Thor-tech-of-metal) [reply charts/ccpay-callback-function/values.preview.template.yaml]: DONE

File: charts/ccpay-callback-function/Chart.yaml (hunk 1 above)
Comment (Thor-tech-of-metal) [reply charts/ccpay-callback-function/Chart.yaml]: DONE

File: charts/ccpay-callback-function/values.preview.template.yaml [hunk 3]
```diff
@@ -5,19 +5,20 @@ function:
...
   resourceGroup: fees-pay-rg
+  sbNamespace: "fees-pay-servicebus-preview"
   teamName: "Fees Pay"
   location: ukwest
   serviceplan: standard
   setup:
     topics:
-      - name: servicecallbacktopic
-        subscriptionNeeded: yes
+    - name: servicecallbacktopic
```
Comment (timja) [inline charts/ccpay-callback-function/values.preview.template.yaml:22]: > * NamespacesTopicsSubscription.servicebus.azure.com "ccpay-callback-function-pr-98-servicebus-servicecallbacktopic" is invalid: spec.azureName: Too long: may not be longer than 50

Not sure if you need to change anything else but maybe you could do:

```suggestion
    - name: callback
```

**Pull request merged**